api/v1alpha1/jobset_types.go

```diff
@@ -22,39 +24,54 @@ import (
 
 // JobSetSpec defines the desired state of JobSet
 type JobSetSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
+	// Jobs is the group of jobs that will form the set.
+	Jobs []JobTemplate `json:"jobs"`
 
-	// Foo is an example field of JobSet. Edit jobset_types.go to remove/update
-	Foo string `json:"foo,omitempty"`
+	// If sequentialStartup if set, then spec.jobs will be created one at a time
+	// according to their order in the list. A job is created only after
+	// the pods of the previous one are Ready.
+	SequentialStartup *bool `json:"sequentialStartup,omitempty"`
 }
 
 // JobSetStatus defines the observed state of JobSet
 type JobSetStatus struct {
-	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
+	// A list of pointers to currently running jobs.
+	// +optional
+	Active []corev1.ObjectReference `json:"active,omitempty"`
```

Comment:
This shouldn't be necessary, the jobs created by the jobset should have ownerReference set that points to the jobset instance they belong to; we should also add a selector as well that we automatically populate, but we can leave that for later.

Reply:
Ok, I removed it.